examples/create_redis_basic_instance/providers.tf

```diff
@@ -0,0 +1,31 @@
+# Do not alias this provider - it seems that Terraform or Google requires at least
+# one 'google' provider without an alias, otherwise it complains about the 'google'
+# provider being missing.
+
+provider "google" {
+  project     = var.google_project
+  region      = var.google_region
+  credentials = var.google_credentials
+
+  scopes = [
```

Comment:
Most of these aren't needed.

Reply:
I duplicated the existing test-case and only updated for the conditions I wanted to test. It could be worthwhile addressing the legacy cruft, although I would prefer to tackle that in a follow-up PR so the downstream work for rightsizing Redis can start, @MXfive.

Reply:
[Backlog ticket](https://linear.app/honestbank/issue/DEVOP-4850/terraform-gcp-memorystore-examples-require-cleaning-up)